### DISCUSSION TITLE
Incorrect go package version when there are multiple in a repo (go submodules)

### DISCUSSION THREAD

**Discussion by jdechicchis** [renovatebot/renovate#9511] [Request Help] [answered]
[Based on this ](https://github.com/go-modules-by-example/index/blob/master/009_submodules/README.md) the following is a valid go project structure:
```
|-- go.mod
|-- b
|   |-- go.mod
|   `-- b.go
`-- a
    |-- go.mod
    `-- a.go
```

To tag `a` and `b` with different versions you might create tags `a/v1.0.0` and `b/v1.1.0`. However, renovate seems to incorrectly think the latest version of `a` is `v1.1.0` in some cases. Was curious if anyone else has run into this issue. I'm running renovate against a private GitLab repo but happy to provide a repro using a public GitHub repo

**Comment by rarkins** [comment, marked as answer]
I think same problem as in https://github.com/renovatebot/renovate/issues/9370 ?

**Reply by jdechicchis** [reply]
thank you for the quick reply! seems to be the same issue
